docs/database/database-metrics.md

```diff
@@ -35,6 +37,132 @@ and attributes but more may be added in the future.
 > until a transition plan to the (future) stable semantic conventions has been published.
 > Conventions include, but are not limited to, attributes, metric and span names, and unit of measure.
 
+## Database operation
+
+### Metric: `db.client.operation.duration`
+
+**Status**: [Experimental][DocumentStatus]
+
+This metric is [required][MetricRequired].
+
+When this metric is reported alongside a database operation span, the metric value SHOULD be the same as the database operation span duration.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[ 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 ]`.
```

Comment:
thoughts on reducing the number of buckets based on discussion in #316?

consistency with HTTP client semconv is definitely a good option

but I think we could also make an argument that granularity is more important for measuring service response time (local root spans), and less important for CLIENT calls

(and we can make the argument that HTTP CLIENT calls have buckets consistent with HTTP SERVER calls)

and so maybe we could reduce database buckets to

`[ 0.005, 0.01, 0.05, 0.1, 0.5, 1, 5, 10 ]`

could also consider removing 5 and 10 second buckets? or adding 0.001 second bucket

Reply:
from SIG meeting, consensus on adding `0.001`

for 5, 10 second buckets - complex joins against large tables can take a long time

Reply:
Updated boundaries per this discussion.